docs/docsite/rst/getting_started_ee/build_execution_environment.rst

```diff
@@ -58,3 +58,8 @@ You can also use Ansible Navigator to view detailed information about the image.
 Run the ``ansible-navigator`` command, type ``:images`` in the TUI, and then choose ``postgresql_ee``.
 
 Proceed to :ref:`Running your EE<running_execution_environments>` and test the EE you just built.
+
+.. seealso::
+
+  `Running a local container registry for EE <https://forum.ansible.com/t/running-a-local-container-registry-for-execution-environments/206>`_
```

Comment:
Hmm. I'm not too sure about this one because it's a forum post rather than docs. I don't know if we have a policy on it or not but I feel like the "see also" section should really be for `ref` links to other docs.

Reply:
fwiw we don't have a policy..erm... for any links. So I can't say how often there are links to non-docs entities, inside or outside the see also sections.  I can say some see also's have non :ref: links - that is direct urls to other sites.

My guess is we have blog post links in the docs 'somewhere'.  Which to me is similar to this forum post. So how do we proceed from here?

I see three options:
1 - allow this PR to go through as it's useful information
2 - Copy the forum post to some 'official docs' somewhere.  This is where we hit the snag that there isn't a home yet for this kind of detail.
3 - Put this PR into WIP and debate the overall docs strategy wrt forum posts, blog posts, and any urls that aren't to official docs somewhere.

For #1 - the problem would be twofold:
a - will the url 'break' at some point and we don't notice?
b - will the content in the forum post go 'stale' and not really be relevant anymore?  Again, we likely have other places where we point to stale content. To me, the driving factor is the content is useful today and if someone does notice in the future that this particular see also link points to stale details, they can open an issue and let us know (or post on the forum and again... let us know.

Or alternately, we could make the forum post a wiki so it's easier for others to update it if it starts to go stale.

Reply:
I'm for Option 1 and the post was already wiki (anyone can edit) but I've just changed it back to forbid editing:) readers can comment if something doesn't work.

Reply:
Based on our[ informal poll ](https://forum.ansible.com/t/what-kind-of-links-should-we-allow-in-ansible-documentation/370/12?u=samccann)in the forum (which, yes, could be a biased vote lol) - people are in favor of allowing forum links for how-to (aka not to any random post, but the semi-formal how to guides etc).